### PR TITLE
DOCS-1605 Code Samples: Using Event Handlers

### PR DESCRIPTION
**Pull request by cameron-white-janrain**

### CODE REVIEW

File: eventHandlers.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+### How do I get the access token generated for a user after registration or login?
+
+```javascript
+janrain.events.onCaptureLoginSuccess.addHandler(function(response) {
+  var accessToken = response.accessToken;
+})
+```
+
+### How do I get a user's display name from localStorage after they have logged in?
```
Comment (kris-janrain) [inline eventHandlers.md]: after the user has

File: eventHandlers.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+```javascript
+janrain.events.onCaptureLoginSuccess.addHandler(function(response) {
+  var janrainCaptureProfileData = localStorage.janrainCaptureProfileData;
+  var janrainCaptureProfileDataParsed = JSON.parse(janrainCaptureProfileData);
+  var displayName = janrainCaptureProfileDataParsed.displayName;
+})
+```
+
+### onCaptureSaveSuccess fires when a user updates anything in their record, but I want to show a specific message when a user updates their password. How can I do that?
```
Comment (kris-janrain) [inline eventHandlers.md]: `onCaptureSaveSuccess` fires

updates the user's password

File: eventHandlers.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+janrain.events.onCaptureSaveSuccess.addHandler(function(response) {
+  var screen = response.screen;
+  var statusMessage = response.statusMessage;
+  if (screen === "changePassword" && statusMessage === "profileSaved") {
+    $('#editProfile .capture_form_item').lastChild.textContents = "your custom message";
+  }
+})
+```
+
+### When an existing user logs in with Twitter for the first time, they get an error message on the Almost done screen when they try to fill in the email address that they previously registered with. How do I display a different error message to these users when the "unique" validation error message is displayed suggesting that they go back and log in with their other credentials?
```
Comment (kris-janrain) [inline eventHandlers.md]: first time, the user gets
the **Almost Done** screen when the user tries to fill in the email addres that the user previously
the `unique` validation
that the user go back and log in with the user's other credentials?

**Comment by Janrain-Colin** [comment]
Looks to me like Kris got them all.  👍 when everything's fixed.

**Pull request merged**